specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/managedClusters.json

```diff
@@ -1308,6 +1402,25 @@
             "type": "string"
           },
           "description": "(PREVIEW) Availability zones for nodes. Must use VirtualMachineScaleSets AgentPoolType."
+        },
+        "enableNodePublicIP": {
+          "type": "boolean",
+          "description": "Enable public IP for nodes"
+        },
+        "scaleSetPriority": {
+          "$ref": "#/definitions/ScaleSetPriority",
+          "description": "ScaleSetPriority to be used to specify virtual machine scale set priority. Default to regular."
+        },
+        "scaleSetEvictionPolicy": {
+          "$ref": "#/definitions/ScaleSetEvictionPolicy",
+          "description": "ScaleSetEvictionPolicy to be used to specify eviction policy for low priority virtual machine scale set. Default to Delete."
+        },
+        "nodeTaints": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Taints added to new nodes during node pool create and scale."
```

Comment:
Worth describing the format of a 'taint' in the description?

Reply:
Good idea. Added an example taints taking from official kubernetes documentation